ow_lander/scripts/mechanical_power_arm.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import rospy
+from sensor_msgs.msg import JointState
+from std_msgs.msg import Float64
+
+# Python libs
+import sys, time
+import numpy as np
+
+#history = []
+
+VERBOSE=True
+
+class MechanicalPower:
+    def __init__(self):
+        '''Initialize ros publisher, ros subscriber'''
+        # topic where we publish
+        self.power_pub = rospy.Publisher('/mechanical_power/raw', Float64, queue_size = 10)
+        self.power_pub_a = rospy.Publisher('/mechanical_power/average', Float64, queue_size = 10)
+        
+        self.subscriber = rospy.Subscriber('/joint_states', JointState, self.callback, queue_size = 1)
+        self.history = []
```

Comment:
I recommend using a _deque_ instead of a _list_, the **history** data member here grows indefinitely. It may not pose a real problem since the data size but things add up.

Reply:
Thanks for the suggestion. I will do this later. I will have to get used to deque.